cluster-state-service/internal/features/e2etasksteps/common_steps.go

```diff
@@ -29,7 +29,7 @@ type Exception struct {
 
 var (
 	// Lists to memorize results required for the subsequent steps
-	ecsTaskList   = []ecs.Task{}
+	EcsTaskList   = []ecs.Task{}
```

Comment:
It's not just a variable name. I'm using the state assigned to EcsTaskList in the task steps within the instance steps, due to the instance streaming steps being dependent on the task steps. Since the instance/task steps are already coupled, I don't think it's worth the effort to completely refactor this variable usage.